File: Aula_0527/lista03ex02.cs
using System;

class Frete {
  private double distancia, peso;
  public Frete(double distancia, double peso) {
    this.distancia = distancia;
    this.peso = peso;
  }
  public double GetDistancia() {
    return distancia;
  }
  public double GetPeso() {
    return peso;
  }
  public void SetDistancia(double distancia) {
    this.distancia = distancia;
  }
  public void SetPeso(double peso) {
    this.peso = peso;
  }
  public double CalcFrete() {
    return 0.01 * distancia * peso;
  }  
  public override string ToString() {
    return $"Distância = {distancia} km, Peso = {peso} kg";
  }
}

class Program {
  public static void Main() {
    Frete f = new Frete(100, 10);
    Console.WriteLine(f);
    Console.WriteLine(f.GetDistancia());
    Console.WriteLine(f.GetPeso());
    Console.WriteLine(f.CalcFrete());
    f.SetDistancia(150);
    Console.WriteLine(f);
    Console.WriteLine(f.CalcFrete());
  }
}

File: Aula_0527/lista03ex02.java
class Frete {
  private double distancia, peso;
  public Frete(double distancia, double peso) {
    this.distancia = distancia;
    this.peso = peso;
  }
  public double getDistancia() {
    return distancia;
  }
  public double getPeso() {
    return peso;
  }
  public void setDistancia(double distancia) {
    this.distancia = distancia;
  }
  public void setPeso(double peso) {
    this.peso = peso;
  }
  public double calcFrete() {
    return 0.01 * distancia * peso;
  }  
  public String toString() {
    return String.format("Distância = %f km, Peso = %f kg", distancia, peso);
  }
}

class Program {
  public static void main(String[] args) {
    Frete f = new Frete(100, 10);
    System.out.println(f);
    System.out.println(f.getDistancia());
    System.out.println(f.getPeso());
    System.out.println(f.calcFrete());
    f.setDistancia(150);
    System.out.println(f);
    System.out.println(f.calcFrete());
  }
}